src/ValidationData.php

```diff
@@ -24,24 +24,30 @@ class ValidationData
      */
     private $items;
 
+    /**
+     * The leeway (in seconds) to use when validating time claims
+     * @var int
+     */
+    private $leeway;
```

Comment:
I was wondering if it makes sense to have this as a property... I mean, it's only used for configuring the current time, maybe it would be better to simply pass it to `ValidationData#setCurrentTime()`... @Ocramius any thoughts?

Reply:
I'm completely fine with doing it either way. Leaving it as it is makes sense if it's fairly common to use the `setCurrentTime()` method 1 or more times after initial construction of the object while keeping the same leeway. However, if users would want to keep changing the leeway without constructing a new object, or if they rarely use `setCurrentTime()` directly at all, then passing leeway as a parameter to `setCurrentTime()` makes more sense.

Reply:
@m777z I'm have no clue on how users are using this object, and since it will be deprecated very soon I don't think we should debate this much. Let's go with your suggested approach and if needed we change it. Thanks!

Reply:
@lcobucci I am assuming you mean it is okay to just leave things as they are for this part. If so, are there other changes I need to make before this pull request is merged?

Reply:
@m777z that's correct. There're some small CS changes (which I'll handle ASAP), I just didn't manage to work on this. Plan to work on it tomorrow